crates/bevy_reflect/Cargo.toml

```diff
@@ -25,6 +25,7 @@ debug_stack = []
 documentation = ["bevy_reflect_derive/documentation"]
 # Enables function reflection
 functions = ["bevy_reflect_derive/functions"]
+alloc = []
```

Comment:
to keep everything as it was, this should be part of the default feature of the bevy_reflect crate

Reply:
done